scipy/optimize/_lsq/least_squares.py

```diff
@@ -638,6 +639,7 @@ def least_squares(
     estimate it by finite differences and provide the sparsity structure of
     Jacobian to significantly speed up this process.
 
+    >>> import numpy as np
```

Comment:
I think this import statement is not needed because it is already imported at the top of example.

Reply:
I'll take note of this.